spec/requests/legalaid_application_spec.rb

```diff
@@ -10,4 +13,27 @@
       expect(response.status).to eql(201)
     end
   end
+
+  describe 'GET /v1/applications/:id' do
+    before do
+      stub_request(:get, '/v1/applications/' + application_ref)
+        .to_return(status: 200, body: [{
+          "id": 2,
+          "name": applicant_name,
+          "date_of_birth": applicant_date_of_birth,
+          "created_at": '2018-08-26T22:47:06.093Z',
+          "updated_at": '2018-08-26T22:47:06.093Z',
+          "application_ref": application_ref,
+          "applicant_id": 2
+        }].to_json)
+    end
+
+    it 'returns a complete application' do
+      get '/v1/applications/' + application_ref
```

Comment:
I’m guessing you’re wanting to test the logic in `application_controller.rb`. You shouldn’t need the `stub_request`. 

`stub_request` is used to mock out HTTP requests. Useful if your production code is making calls to an API. E.g. if we needed to make an API request to a DWP service.

You would need to have an Applcant and LegalAidApplication record in place to drive your current production code.